client/src/app/videos/+video-watch/video-watch.component.html

```diff
@@ -21,14 +21,17 @@
               {{ video.channel.displayName }}
             </a>
             <!-- Here will be the subscribe button -->
-            <my-help helpType="custom" customHtml="You can subscribe to this account via any ActivityPub-capable fediverse instance. For instance with Mastodon or Pleroma you can type in the search box <strong>@{{video.account.displayName}}@{{video.account.host}}</strong> and subscribe there. Subscription as a PeerTube user is being worked on in <a href='https://github.com/Chocobozzz/PeerTube/issues/470'>#470</a>."></my-help>
+            <my-help helpType="custom" customHtml="You can subscribe to this account via any ActivityPub-capable fediverse instance. For instance with Mastodon or Pleroma you can type in the search box <strong>@{{video.account.name}}@{{video.account.host}}</strong> and subscribe there. Subscription as a PeerTube user is being worked on in <a href='https://github.com/Chocobozzz/PeerTube/issues/470'>#470</a>."></my-help>
           </div>
 
           <div class="video-info-by">
-            <a [routerLink]="[ '/accounts', video.by ]" title="Go the account page">
+            <a [routerLink]="[ '/accounts', video.by ]" title="Go to the account page">
               <span>By {{ video.by }}</span>
               <img [src]="video.accountAvatarUrl" alt="Account avatar" />
             </a>
+            <a *ngIf="!video.isLocal" href="https://{{video.account.host}}" title="Go to the instance of origin of this account">
+              <span>on {{ video.account.host }}</span>
```

Comment:
The info is already in the `video.by`. Please remove this block.

If you want to have a link directly in the origin instance, put it in https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/app/%2Baccounts/accounts.component.html#L10